### DISCUSSION TITLE
Sierra EM7345 for Lenovo Thinkpad T560

### DISCUSSION THREAD

**Discussion by anishjp** [danielewood/sierra-wireless-modems#79] [Q&A]
Hi Guys

Does anyone know if a Sierra EM7345 will work on a Thinkpad T560 or has someone managed to get it working on a T560?

According to Lenovo, T560 supports only EM7455.

Regards,
Anish

**Comment by tmuehlhoff** [comment]
you may need to patch your BIOS to Whitelist other modems. This

https://www.bios-mods.com/forum/archive/index.php?thread-32055.html
https://www.bios-mods.com/forum/archive/index.php?thread-22453.html

http://paranoid.anal-slavery.com/biosmods/skylake.html

**Reply by anishjp** [reply]
Thanks for your reply.

I bought a EM7455 and made it working in my T560.
